dolphinscheduler-ui/src/components/chart/index.ts

```diff
@@ -44,6 +54,24 @@ function initChart<Opt extends ECBasicOption>(
   }
 
   const resize = throttle(() => {
+    if (type === 'gauge') {
```

Comment:
Why only gauge needs to adapt?

Reply:
will approve CI run when we solve this discuss